### PR TITLE
sleuthkit: new package

### PR DESCRIPTION
**Pull request by ifurther**
#5089

I just add build.sh file

### CODE REVIEW

File: packages/sleuthkit/build.sh [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Skeleton build.sh script for new package.
+# For reference about available fields, check the Termux Developer's Wiki page:
+# https://github.com/termux/termux-packages/wiki/Creating-new-package
+
```
Comment (xeffyr) [inline packages/sleuthkit/build.sh:4]: Remove these comments

File: packages/sleuthkit/build.sh [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+TERMUX_PKG_HOMEPAGE=https://sleuthkit.org/
+TERMUX_PKG_DESCRIPTION="The Sleuth Kit® (TSK) is a library and collection of command line digital forensics tools that allow you to investigate volume and file system data. "
+TERMUX_PKG_LICENSE="MIT and BSD 3-clause" #have other
```
Comment (xeffyr) [inline packages/sleuthkit/build.sh:7]: Separate license names with comma. Other variants are not supported by parser.

File: packages/sleuthkit/build.sh [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+TERMUX_PKG_VERSION=4.9.0
+TERMUX_PKG_SRCURL=https://github.com/sleuthkit/sleuthkit/archive/sleuthkit-${TERMUX_PKG_VERSION}.tar.gz
+TERMUX_PKG_SHA256=00faf25c8ffd9271b1c144586a4197fa3ed4cefa70adab1d73ad39376ae009fe
+TERMUX_PKG_DEPENDS="libsqlite, ncurses"
+TERMUX_PKG_BUILD_IN_SRC=true
+
+
+
+termux_step_install_license() {
+        for filename in $TERMUX_PKG_SRCDIR/licenses/*; do
```
Comment (xeffyr) [inline packages/sleuthkit/build.sh:17]: This step is not needed as soon as licenses are standard.

**Comment by ifurther** [comment]
@xeffyr I update if.

**Comment by ifurther** [comment]
@xeffyr thank you

**Pull request merged**